File: LuaPythonWorkSessons/WorkSesson7Task4.lua
ShoppingList = {
    "Water",
    "Chicken",
    "Rice",
    "Laptop",
    "Gas"
}

ItemToCheck = {
    "Water",
    "Money",
    "Rice",
    "Bottle",
    "Chicken",
    "Gas"
}

for Key, Value in ipairs(ShoppingList) do
    for k, v in ipairs(ItemToCheck) do
        if Value == v then
            print("Your items to check has "..v)
        end
    end
end

File: LuaPythonWorkSessons/WorkSesson7Task4.py
ShoppingList = [
    "Bottle",
    "Gas",
    "Chicken",
    "Rice",
    "Meat"
]

ItemList = [
    "HotDog",
    "Water",
    "Gas",
    "Oxygen",
    "Power",
    "Meat"
]

for Item in ShoppingList:
    for List in ItemList:
        if Item == List:
            print(f"You have {List} in your itemlist thats also on the shopping list")